spec/rental_spec.rb

```diff
@@ -0,0 +1,18 @@
+require_relative '../rental'
+
+describe Rental do
+  it 'something' do
+    date = '2022-04-12'
+    person = double('person')
+    book = double('book')
+
+    allow(person).to receive(:name) { 'Robert' }
+    allow(book).to receive(:title) { 'The Sun' }
+
+    rental = Rental.new(date, person, book)
+    expect(rental).to be_an_instance_of(Rental)
+    expect(rental.book).to eql(book)
+    expect(rental.person.name).to eql('Robert')
+    expect(rental.book.title).to eql('The Sun')
+  end
```

Comment:
- [ ] Since we are aiming for a 100% test coverage, it will be nice to add tests all the methods in all the classes. That said, it will be nice to add a test for the `add_rentals` method, something like: 
```ruby
  context "When adding a new rentals" do

    it "The add_rental method returns a rental and adds its to the book" do
      book = Book.new("Title", "Author")
      person = Person.new(24, "Mih")

      rental = person.add_rental(book, "2022-03-04")

      expect(person.rentals).to eql [rental]
    end
  end
```

Please let's cover all our methods with tests.